### ISSUE TITLE
Are this model robust to differnet 3DVAE？ If I change to use other better 3DVAE and pretrain for two weeks, will it work?

### ISSUE THREAD

**Issue by huangjch526** [Alpha-VLLM/Lumina-Video#7]
Thanks and the authors are very good man

**Comment by ChrisLiu6** [comment]
I'm not sure... All I know is:

1. Can we change the vae in a zero-shot manner?
No, unless the new vae has the same encoder as the current one but improved decoder, similar to the vae improvement from SD1.5 to SD2.1

2. Can we finetune the model on a different vae?
We have indeed  made such trials in our early text-to-image experiments. Unfortunately, our observation was that such cross-vae initialization is worse than simple random initialization as reflected by loss curve. On the other had, we have also noticed that some existing works have adopted cross-vae initialization, so maybe in different situations the conclusion would differ.
